app/controllers/csvs_controller.rb

```diff
@@ -0,0 +1,54 @@
+class CsvsController < ApplicationController
+
+  def index
+    respond_to do |format|
+      @csv_exports = CsvExport.where(user_id: current_user.id)
+      format.html
+      format.json { render json: @csv_exports }
+    end
+  end
+  
+  def show
+    respond_to do |format|
+      @csv_export = CsvExport.find(params[:id])
+      format.html
+      format.json { render json: @csv_export }
+      format.csv { download(params) }
+    end
+  end
+
+  def create
+    csv_export = CsvExport.create!(user: current_user, content: params[:content], filters: params.to_json)
```

Comment:
storing all of params includes actions/contoller etc unnecessary things, can we store `params[:filter]` instead ?